components/SearchBox/index.js

```diff
@@ -19,14 +17,14 @@ var SearchBox = React.createClass({
 
     return (
       <input type="text"
-      placeholder={this.props.placeholder}
-      name="algolia-query"
-      className={classNames}
-      data-role="autocomplete"
-      autoComplete="off"
-      autoFocus="autofocus"
-      onChange={this.change}
-      role="textbox" />
+        placeholder={this.props.placeholder}
+        name="algolia-query"
+        className={classNames}
+        data-role="autocomplete"
```

Comment:
I would add this only for autocomplete.js where the suggestions are only dependent of the search input. Whereas in a an instant ui the suggestion are not dedicated to the search input but to all the interface and filters. We need an accessibility expert that will review theses questions for days :D